pootle/checks.py

```diff
@@ -340,6 +340,81 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W019",
                 ))
 
+    try:
+        edit_coef = settings.POOTLE_SCORING_CONF['EDIT_COEF']
+        try:
+            edit_coef + 0.0
+        except TypeError:
+            errors.append(checks.Critical(
+                _("Invalid value for EDIT_COEF in POOTLE_SCORING_CONF."),
+                hint=_("Set a valid value for EDIT_COEF "
+                       "in POOTLE_SCORING_CONF."),
+                id="pootle.C015",
+            ))
+    except KeyError:
+        errors.append(checks.Critical(
+            _("POOTLE_SCORING_CONF has no EDIT_COEF."),
+            hint=_("Set an EDIT_COEF in POOTLE_SCORING_CONF."),
+            id="pootle.C014",
+        ))
+
+    try:
+        review_coef = settings.POOTLE_SCORING_CONF['REVIEW_COEF']
+        try:
+            review_coef + 0.0
+        except TypeError:
+            errors.append(checks.Critical(
+                _("Invalid value for REVIEW_COEF in POOTLE_SCORING_CONF."),
+                hint=_("Set a valid value for REVIEW_COEF "
+                       "in POOTLE_SCORING_CONF."),
+                id="pootle.C017",
+            ))
+
+    except KeyError:
+        errors.append(checks.Critical(
+            _("POOTLE_SCORING_CONF has no REVIEW_COEF."),
+            hint=_("Set an REVIEW_COEF in POOTLE_SCORING_CONF."),
+            id="pootle.C016",
+        ))
+
+    try:
+        sugg_coef = settings.POOTLE_SCORING_CONF['SUGG_COEF']
+        try:
+            sugg_coef + 0.0
+        except TypeError:
+            errors.append(checks.Critical(
+                _("Invalid value for SUGG_COEF in POOTLE_SCORING_CONF."),
+                hint=_("Set a valid value for SUGG_COEF "
+                       "in POOTLE_SCORING_CONF."),
+                id="pootle.C019",
+            ))
```

Comment:
I think it is better to get this second `try` clase out of the parent `try`. You can do this with:

```python
    try:
        sugg_coef = settings.POOTLE_SCORING_CONF['SUGG_COEF']
    except KeyError:
        ...
    else:
        try:
            sugg_coef + 0.0
        except TypeError:
            ...
```

IMHO this leads to cleaner code where you can easily spot what is being tested on each `try` clause.